openjdk.build/docs/build.md

```diff
@@ -37,7 +37,7 @@ These prereqs must be installed before attempting to build openjdk-systemtest
 | Windows Sysinternals  | https://technet.microsoft.com/en-us/sysinternals/bb469936.aspx| stf.core                 | Download from https://download.sysinternals.com/files/SysinternalsSuite.zip                                                                                                                                                                                | Unzip to PREREQS_ROOT/windows_sysinternals                                                                                                                                                                                                                            | Yes                                |
 | CVS client            | https://netbeans.org/cddl-gplv2.html                          | openjdk.test.mauve       | Download from https://netbeans.org/projects/versioncontrol/downloads/download/org-netbeans-lib-cvsclient.jar                                                                                                                                               | Place in PREREQS_ROOT/cvsclient                                                                                                                                                                                                                                       | Yes                                |
-| wget                  | https://www.gnu.org/copyleft/gpl.html                         | stf.build, openjdk.build | Windows - download from https://sourceforge.net/projects/gnuwin32/files/wget/1.11.4-1/wget-1.11.4-1-bin.zip                                                                                                                                                | Add to PATH                                                                                                                                                                                                                                                           | No                                 |
+| wget                  | https://www.gnu.org/copyleft/gpl.html                         | stf.build, openjdk.build | Windows - download from https://sourceforge.net/projects/gnuwin32/files/wget/1.11.4-1/wget-1.11.4-1-bin.zip<br>This needs below dependency so download and add this to PATH.<br>https://sourceforge.net/projects/gnuwin32/files/wget/1.11.4-1/wget-1.11.4-1-dep.zip                                                                                                                                                | Add to PATH                                                                                                                                                                                                                                                           | No                                 |
```

Comment:
Do the files really need to be on the PATH?  On my local machine they are just in the same directory as wget.exe.